testvectors/example.go

```diff
@@ -31,38 +25,32 @@ type:
   primitiveAssociative: true
 `)
 
-	var tests []*Vector
-	err := yaml.Unmarshal([]byte(`
-- name: list insert 1
-  schemaName: listordering-primitiveInlineList
-  lastObject: |
-      items:
-      - a
-      - b
-      - c
-  liveObject: |
-      items:
-      - a
-      - b
-      - c
-      - d
-  newObject: |
-      items:
-      - a
-      - b
-      - e
-      - c
-  expectedObject: |
-      items:
-      - a
-      - b
-      - e
-      - c
-      - d
-`), &tests)
-	if err != nil {
-		panic(fmt.Sprintf("test cases defined wrong: %v", err))
+	exampleSequence := &Sequence{
```

Comment:
Maybe we should drop the sequence concept completely and make this look more like the example test?

Reply:
Oh sorry, this is the old one and I forgot to delete. So everything in testvectors would be thrown away for now in favor of tests/...